lib/pos_epoch.go

```diff
@@ -14,7 +14,10 @@ import (
 
 type EpochEntry struct {
 	EpochNumber      uint64
-	FinalBlockHeight uint64
+	StartBlockHeight uint64
```

Comment:
Just a nit: Matt Foley and I went back and forth a long time ago on when to use Start/End/Initial/Final. We generally preferred Initial/Final since they're less ambiguous that the heights form an inclusive range.
```suggestion
	InitalBlockHeight uint64
```